src/DataStructures/VectorImpl.hpp

```diff
@@ -557,16 +557,16 @@ std::ostream& operator<<(std::ostream& os,
  * \brief Defines the overloaded elementwise binary function OPERATOR
  *  taking INTYPEL and INTYPER, returning OUTTYPE (vector types)
  */
-#define DEFINE_VECTOR_ELEMENTWISE_BINARY_FUNCTION(INTYPEL, INTYPER, OUTTYPE,   \
-                                                  OPERATOR)                    \
-  OUTTYPE OPERATOR(const INTYPEL& l, const INTYPER& r) noexcept {              \
-    ASSERT(l.size() == r.size(),                                               \
-           "Elementwise operations must be between objects of the same size"   \
-               << " not " << l.size() << " and " r.size());                    \
-    OUTTYPE ret{l.size()};                                                     \
-    std::transform(l.begin(), l.end(), r.begin(), ret.begin(),                 \
+#define DEFINE_VECTOR_ELEMENTWISE_BINARY_FUNCTION(INTYPEL, INTYPER, OUTTYPE, \
```

Comment:
this commit does nothing but whitespace changes, not what the commit message says